pkg/kubelet/cm/container_manager.go

```diff
@@ -60,6 +60,9 @@ type ContainerManager interface {
 	// GetNodeAllocatable returns the amount of compute resources that have to be reserved from scheduling.
 	GetNodeAllocatableReservation() v1.ResourceList
 
+	// GetNodeCapacity returns the amount of compute resources tracked by container manager available on the node.
+	GetCapacity() v1.ResourceList
+
```

Comment:
My understanding is that currently in container manager initialization, it scans the system resources and keeps a cache of the capacity. 

As for `HasDedicatedImageFs`, it seems like the only place it is called in during container manager initialization.  Would it make more sense for it to just be a private function then?